net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=@(PACKAGE_wireless-tools||PACKAGE_iw) +ethtool +procps +CONFIG_USB_SUPPORT:usbutils +CONFIG_PCI_SUPPORT:pciutils
```

Comment:
In the reality of linux today, is there any advantage to even allowing wireless-tools||iw?  If I had to guess, everyone requires iw anyway, it may just be best to depend on iw and not at all on wireless-tools.  airmon-ng needs one or the other, but everything about wireless-tools is deprecated anyway.